stable/postgresql/templates/statefulset-slaves.yaml

```diff
@@ -52,7 +52,7 @@ spec:
       {{- if .Values.terminationGracePeriodSeconds }}
       terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
       {{- end }}
-      {{- if and .Values.persistence.enabled .Values.securityContext.enabled}}
+      {{- if and .Values.volumePermissions.enabled .Values.persistence.enabled .Values.securityContext.enabled }}
```

Comment:
Now that we have a specific switch for the initContainer, I would delete `.Values.securityContext.enabled` from the if condition so we cover the possible combinations.

Reply:
Does it make sense to do `chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }} /bitnami` if `.Values.securityContext.enabled=false`?

Reply:
Yes, well, what It makes sense is to run:

```
chown -R 1001:1001 /bitnami
```
as it is the user running the PostgreSQL container.

Reply:
Okay.. Those are the default values. Let's make it.